src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7996,6 +7996,15 @@ namespace UnitarySystems {
                 }
                 this->m_DesiredOutletTemp = min(this->m_DesiredOutletTemp, MaxOutletTemp);
 
+                if (this->m_FanOpModeSchedPtr > 0) {
+                    if (ScheduleManager::GetCurrentScheduleValue(this->m_FanOpModeSchedPtr) == 0.0) {
+                        this->m_FanOpMode = DataHVACGlobals::CycFanCycCoil;
+                    } else {
+                        this->m_FanOpMode = DataHVACGlobals::ContFanCycCoil;
+                        DataHVACGlobals::OnOffFanPartLoadFraction = 1.0;
+                    }
+                }
+
```

Comment:
Well, this works but now this same code is in 2 places, here and in initLoadBasedControl. Shouldn't these get moved to initUnitarySystems inside the FirstHVACIteration test (line 1084) so that both control types execute this, it's only done once per time step and there is only 1 copy of these lines of code.

Reply:
That makes sense, I'll make this adjustment once the CI finishes.  Thanks